c2pa/__init__.py

```diff
@@ -11,8 +11,35 @@
 # specific language governing permissions and limitations under
 # each license.
 
-from .c2pa_api import Reader, Builder, create_signer, create_remote_signer, sign_ps256
-from .c2pa import Error, SigningAlg, CallbackSigner, sdk_version, version
-from .c2pa.c2pa import _UniffiConverterTypeSigningAlg, _UniffiConverterTypeReader, _UniffiRustBuffer
+from .c2pa_api import (
+    Reader,
+    Builder,
+    create_signer,
+    create_remote_signer,
+    sign_ps256,
+    load_settings_file,
+)
+from .c2pa import Error, SigningAlg, CallbackSigner, sdk_version, version, load_settings
+from .c2pa.c2pa import (
```

Comment:
I never saw the syntax with parenthesis before, hope it doesn't have adverse strange side-effects. After researching it, that's allowed only for Python 3.10 and later, so Python-wise should be fine.